<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewWorld</class>
 <widget class="QMainWindow" name="NewWorld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewWorld</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Back Office New World </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetApp">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>178</width>
                <height>17</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAnnulConnect">
              <property name="text">
               <string>Annuler</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonValidConnect">
              <property name="text">
               <string>Valider</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>125</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Login :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Mot de passe :</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLineEdit" name="lineEditLogAdmin">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Admin</string>
                   </property>
                   <property name="readOnly">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditMdpAdmin">
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Les Rayons</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QTableView" name="tableViewRay">
                  <property name="maximumSize">
                   <size>
                    <width>190</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="dragEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="dragDropOverwriteMode">
                   <bool>false</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::InternalMove</enum>
                  </property>
                  <property name="defaultDropAction">
                   <enum>Qt::IgnoreAction</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <property name="gridStyle">
                   <enum>Qt::NoPen</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <spacer name="horizontalSpacer_7">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                        <weight>75</weight>
                        <italic>true</italic>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Ordre :</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonUp">
                    <property name="text">
                     <string>/\</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonDown">
                    <property name="text">
                     <string>\/</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <spacer name="horizontalSpacer_8">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_7">
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                        <weight>75</weight>
                        <italic>true</italic>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Nouveau rayon :</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Désignation :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditDesRay"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonRayValid">
                    <property name="text">
                     <string>Valider</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <spacer name="horizontalSpacer_14">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="font">
                       <font>
                        <pointsize>14</pointsize>
                        <weight>75</weight>
                        <italic>true</italic>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Modification :</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditModifRay"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonRayModif">
                    <property name="text">
                     <string>Modifier</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Les Produits</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QToolBox" name="toolBox">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_3">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>512</width>
                   <height>381</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Actif</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <item>
                       <widget class="QTableView" name="tableViewProdActif">
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="alternatingRowColors">
                         <bool>true</bool>
                        </property>
                        <property name="selectionBehavior">
                         <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QComboBox" name="comboBoxProdAct"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_9">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_7">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_5">
                          <item>
                           <spacer name="horizontalSpacer_5">
                            <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>40</width>
                              <height>20</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_5">
                            <property name="font">
                             <font>
                              <pointsize>14</pointsize>
                              <weight>75</weight>
                              <italic>true</italic>
                              <bold>true</bold>
                             </font>
                            </property>
                            <property name="text">
                             <string>Nouveau produit :</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_6">
                          <item>
                           <widget class="QComboBox" name="comboBoxNewProd">
                            <item>
                             <property name="text">
                              <string>Choix du rayon</string>
                             </property>
                            </item>
                           </widget>
                          </item>
                          <item>
                           <layout class="QHBoxLayout" name="horizontalLayout_4">
                            <item>
                             <widget class="QLabel" name="label_4">
                              <property name="text">
                               <string>Désignation :</string>
                              </property>
                             </widget>
                            </item>
                            <item>
                             <spacer name="horizontalSpacer_10">
                              <property name="orientation">
                               <enum>Qt::Horizontal</enum>
                              </property>
                              <property name="sizeHint" stdset="0">
                               <size>
                                <width>40</width>
                                <height>20</height>
                               </size>
                              </property>
                             </spacer>
                            </item>
                           </layout>
                          </item>
                          <item>
                           <widget class="QLineEdit" name="lineEditProdDesi"/>
                          </item>
                          <item>
                           <widget class="QPushButton" name="pushButtonProdValid">
                            <property name="text">
                             <string>Valider</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="Line" name="line_3">
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_8">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_13">
                          <item>
                           <spacer name="horizontalSpacer_12">
                            <property name="orientation">
                             <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                             <size>
                              <width>40</width>
                              <height>20</height>
                             </size>
                            </property>
                           </spacer>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_10">
                            <property name="font">
                             <font>
                              <pointsize>14</pointsize>
                              <weight>75</weight>
                              <italic>true</italic>
                              <bold>true</bold>
                             </font>
                            </property>
                            <property name="text">
                             <string>Modification : </string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <widget class="QComboBox" name="comboBoxModifProd">
                          <item>
                           <property name="text">
                            <string>Choix du rayon</string>
                           </property>
                          </item>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLineEdit" name="lineEditProdModif"/>
                        </item>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_12">
                          <item>
                           <widget class="QPushButton" name="pushButtonProdDesac">
                            <property name="text">
                             <string>Désactiver</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_4">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>512</width>
                   <height>381</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Inactif</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QTableView" name="tableViewProdDesact">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="alternatingRowColors">
                       <bool>true</bool>
                      </property>
                      <property name="selectionBehavior">
                       <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_10">
                      <item>
                       <widget class="QComboBox" name="comboBoxProdDesact">
                        <property name="minimumSize">
                         <size>
                          <width>155</width>
                          <height>0</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButtonProdReact">
                        <property name="text">
                         <string>Ré-activer</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
